plugins/modules/zos_copy.py

```diff
@@ -2273,6 +2346,37 @@ def run_module(module, arg_def):
         # Copy to sequential data set (PS / SEQ)
         # ---------------------------------------------------------------------
         elif dest_ds_type in data_set.DataSet.MVS_SEQ:
+            if src_ds_type == "USS" and not is_binary:
+                # Before copying into the destination dataset, we'll make sure that
+                # the source file doesn't contain any carriage returns that would
+                # result in empty records in the destination.
+                # Due to the differences between encodings, we'll normalize to IBM-037
+                # before checking the EOL sequence.
+                new_src = conv_path or temp_path or src
+                enc_utils = encode.EncodeUtils()
+                src_tag = enc_utils.uss_file_tag(new_src)
+
+                if src_tag == "untagged":
```

Comment:
Maybe I am missing it but where are you getting the "untagged", i see that enc_utils.uss_file_tag(..) returns 'None' or the files encoding (tag). Can you help me out if I am missing something.

Reply:
If files aren't tagged and `ls` executes without an abend, the returned tag from the command will be `untagged`, as far as I know, `untagged` is a valid tag.

Reply:
Ah right, you are correct it does come back with `untagged` , not used to seeing that but you are correct.